server/system/schedulers/recordPreviewCleanup.js

```diff
@@ -1,7 +1,7 @@
 const schedule = require('node-schedule')
 
-const SurveyManager = require('../modules/survey/manager/surveyManager')
-const RecordService = require('../modules/record/service/recordService')
+const SurveyManager = require('../../modules/survey/manager/surveyManager')
+const RecordService = require('../../modules/record/service/recordService')
 
```

Comment:
Can we use jus RecordService here ? (i know it was already there)

Reply:
right